core/templates/dev/head/pages/learner_dashboard/LearnerDashboard.js

```diff
@@ -370,10 +370,9 @@ oppia.controller('LearnerDashboard', [
         },
         sort: function(e, ui) {
           /* eslint-disable quote-props */
-          // Reset the position of the window on scrolling. This keeps the mouse
-          // position and elements in sync.
+          // Making top : 0px to avoid irregular change in position.
           ui.helper.css(
-            {'top': ui.position.top + $(window).scrollTop() + 'px'});
+            {'top': '0 px'});
```

Comment:
Have you tested this? I mean this makes the helper dom static, isn't it?

Reply:
Yes, i’ve tested this works fine.
It makes the dom static.